test/integration/controllers/user.spec.js

```diff
@@ -83,11 +86,25 @@ describe('User controller', () => {
     })
   })
 
+  describe(`POST ${endpointUrl}delete`, () => {
+    it('should DELETE users by IDs', async () => {
+      const emails = ['harry@gmail.com', 'hermione@gmail.com']
+      const users =  emails.map( async email => await User.create({ ...testUserForDeleting, email }))
+
+      const usersIds = users.map(user => user._id)
+      
+      const response = await app.post(endpointUrl + 'delete').send(usersIds)
+
+      expect(response.status).toBe(204)
+    })
+  })
+
   describe(`DELETE ${endpointUrl}:id`, () => {
     it('should DELETE user by ID', async () => {
+      
       const response = await app.delete(endpointUrl + testUser._id)
 
-      expect(response.statusCode).toBe(204)
+      expect(response.status).toBe(204)
```

Comment:
why do we have everywhere statusCode but you changed to status? is there any logic\reason behind it?

Reply:
there were some troubles with this test but at this moment i have fixed and changed to statusCode